### PR TITLE
Text and Typography Component

### PR DESCRIPTION
**Pull request by kolanse**

### CODE REVIEW

File: componentLibrary/build.gradle [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.core:core-ktx:1.8.0'
```
Comment (IfyUnoka4) [inline componentLibrary/build.gradle:35]: versions can be extracted

File: componentLibrary/src/main/java/com/example/componentlibrary/atom/text/ToDoTextPreview.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import androidx.compose.ui.unit.dp
+import com.example.componentlibrary.ToDoTypography
+
+@Preview
+@Composable
+fun ToDoTextPreview() {
+    Box(modifier = Modifier.width(400.dp)) {
+        Column {
+            ToDoText(
+                text = "We are Here to Serve you",
```
Comment (IfyUnoka4) [inline componentLibrary/src/main/java/com/example/componentlibrary/atom/text/ToDoTextPreview.kt:19]: string text can be extracted

File: componentLibrary/src/main/java/com/example/componentlibrary/atom/text/ToDoTextPreview.kt [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.componentlibrary.ToDoTypography
+
+@Preview
+@Composable
+fun ToDoTextPreview() {
+    Box(modifier = Modifier.width(400.dp)) {
```
Comment (IfyUnoka4) [inline componentLibrary/src/main/java/com/example/componentlibrary/atom/text/ToDoTextPreview.kt:16]: same with dimensions

Comment (kolanse) [reply componentLibrary/src/main/java/com/example/componentlibrary/atom/text/ToDoTextPreview.kt:16]: Yeah.  Will work on the spacing for this in an upcoming PR.

File: componentLibrary/build.gradle (hunk 1 above)
Comment (kolanse) [reply componentLibrary/build.gradle:35]: There will be a task for it .  Plan is to use build.gradle.it's

**Pull request merged**